app/services/school_update_service.rb

```diff
@@ -0,0 +1,59 @@
+class SchoolUpdateService
+  RB_NAME_MAP = {
+    'Bristol, City of' => 'City of Bristol',
+    'Dorset' => 'Dorset Council',
+    'Herefordshire, County of' => 'Herefordshire',
+    'Kingston upon Hull, City of' => 'Kingston upon Hull',
+  }.freeze
+
+  def update_schools
+    # look at the schools that have changed since the last update
+    last_update = Staging::DataUpdateRecord.last_update_for(:schools)
+
+    # simple updates for schools that are open
+    Staging::School.updated_since(last_update).open.each do |staged_school|
+      school = School.find_by(urn: staged_school.urn)
+      if school
+        update_school(school, staged_school)
+      else
+        create_school(staged_school)
+      end
+    end
+
+    Staging::DataUpdateRecord.updated!(:schools)
+  end
+
+private
+
+  def update_school(school, staged_school)
+    # update school details
+    attrs = staged_attributes(staged_school)
+    school.update!(attrs)
+  rescue ActiveRecord::RecordInvalid => e
+    Rails.logger.error(e.record.errors)
```

Comment:
would it be useful if these were also sent to sentry?

Reply:
related - do we care about updates failing? This is fine if we don't, but I don't know enough about the use case for this to say whether we do or not... @tonyheadford ?

Reply:
Let's skip for now but might be useful later otherwise this will silently fail and we'll miss out on updates